rollup.config.js

```diff
@@ -61,13 +63,17 @@ const plugins = [
       'transform-object-rest-spread',
       'transform-class-properties',
     ],
+    exclude: [
+      'node_modules/**',
+      '*.json',
```

Comment:
Without this, the Karma build fails because babel complains about syntax errors in `package.json`